Makefile

```diff
@@ -398,7 +398,7 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=0.13.0
+protoVer=0.14.0
```

Comment:
We need to tell apps to bump this as well, when they use the proto-builder.
To be honest, still not a fan to have to bump our miniumun go version, while we won't take advantage of the Go 1.21 features and apps can already still build their app with Go 1.21